fiftyone/operators/types.py

```diff
@@ -1777,6 +1777,15 @@ def execute():
             return {"image": "https://voxel51.com/your/image.png"}
 
         def resolve_output(self, ctx):
+            schema = {
+                "height": "100px",
+                "width": "100px",
+                "alt": "My image alt text",
+                "href": "https://voxel51.com",
+                "operator": self.do_something,
```

Comment:
This will only work in a python panel, `resolve_output` is only for `foo.Operator`s.

Reply:
Instead you could say:

```python
"operator": "@my/plugin/my_operator"
```

Reply:
Updated example to make sure Python Panel reference & Operator reference is noted 👍🏽